Tools/Scripts/webkitdirs.pm

```diff
@@ -3218,7 +3234,7 @@ sub runIOSWebKitAppInSimulator($;$)
 
         print "Installing $appBundle.\n";
         # Install custom built app, overwriting an app with the same app identifier if one exists.
-        exitStatus(system("xcrun", "--sdk", "iphonesimulator", "simctl", "install", $simulatedDeviceUDID, $appBundle)) == 0 or die;
+        exitStatus(system(("xcrun", "--sdk", "iphonesimulator", "simctl", "install", $simulatedDeviceUDID, $appBundle))) == 0 or die;
```

Comment:
Interesting that these extra parentheses are needed.

Reply:
Whoops I forgot to get rid of those. I was debugging an issue and I thought I needed to wrap the arguments to make an array. Turns out I just had a newline character at the end of `$appBundle` which is a valid file path character (WHY) and so xcrun failed to find the bundle.